index.html

```diff
@@ -1,149 +1,285 @@
-    <p> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    <button type="button"> Get My Resume </button>
-    <div class="line"></div>
-    <div class="language"> 
-     <div> <img src="img/Rectangle_55.png" alt=""> <h3> Language</h3></div>
-     <ul>
-       <li> Javascript</li>
-       <li> Ruby</li>
-       <li> html</li>
-       <li> Css</li>
-     </ul>
-    </div>
-    <div class="language"> 
-      <div> <img src="img/Rectangle_56.png" alt=""> <h3> Language</h3></div>
-      <ul>
-        <li> Bootstrap</li>
-        <li> Ruby on Rails</li>
-        <li> RSPec</li>
-        <li> CapyBara</li>
-        <li> Selenium</li>
-      </ul>
-     </div>
-     <div class="language"> 
-      <div> <img src="img/Rectangle_57.png" alt=""> <h3> Skills</h3></div>
-      <ul>
-        <li> Codekit</li>
-        <li> Github</li>
-        <li> Codepen</li>
-        <li> Gitlab</li>
-        <li> Terminal</li>
-      </ul>
-     </div>
-  </section>
-</body>
-</html>
\ No newline at end of file
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </form>
+    </section>
+
+    <section class="about-me">
+      <h4>About Me</h4>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button type="button">Get My Resume</button>
+      <div class="line"></div>
+      <div class="language">
+        <div>
+          <img src="img/Rectangle_55.png" alt="" />
+          <h3>Language</h3>
+        </div>
+        <ul>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+      <div class="language">
+        <div>
+          <img src="img/Rectangle_56.png" alt="" />
+          <h3>Language</h3>
+        </div>
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+      <div class="language">
+        <div>
+          <img src="img/Rectangle_57.png" alt="" />
+          <h3>Skills</h3>
+        </div>
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="footer-header">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </div>
+      <div class="footer-form">
+        <form method="post" action="https://formspree.io/f/1"> 
```

Comment:
- [ ] Please ensure that you have registered properly and gotten a valid ID on formspree. At the moment, the form is not been submitted.